src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -895,8 +895,16 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 
         assert(key);
 
-        if (proc_cmdline_key_streq(key, "systemd.gpt_auto") ||
-            proc_cmdline_key_streq(key, "rd.systemd.gpt_auto")) {
+        if (STR_IN_SET(key, "fstab", "rd.fstab")) {
+
+                r = value ? parse_boolean(value) : 1;
+                if (r < 0)
+                        log_warning("Failed to parse fstab switch %s. Ignoring.", value);
```

Comment:
If you'd still like to keep it, then please log the errno here, even when not used in log message.